src/core/src/core_logic/PatchAssessor.py

```diff
@@ -84,6 +84,12 @@ def start_assessment(self):
                 self.telemetry_writer.write_event("Security assessment: " + str(sec_packages), Constants.TelemetryEventLevel.Verbose)
                 self.status_handler.set_package_assessment_status(sec_packages, sec_package_versions, "Security")
```

Comment:
Good to change this to use the Constants entry.

Reply:
Done